docs/http-api.md

```diff
@@ -163,6 +163,26 @@ POST /metrics/delete
 curl -H "X-Org-Id: 12345" --data query=statsd.fakesite.counters.session_start.*.count "http://localhost:6060/metrics/delete"
 ```
 
+
+## Deleting tagged metrics
+
+This will delete the metrics (technically metricdefinitions) matching the `path` parameter(s).
+Note that the data stays in the datastore until it expires.
+Should the metrics enter the system again with the same metadata, the data will show up again.
+
```

Comment:
The soft delete notes were present on this version of the `/metrics/delete` endpoint, but not the `cloud` version, so I've copied that convention here as well.

Reply:
line 170 and 171 should definitely be published in both versions of the docs. this has caused confusion

Reply:
3d457c499802cd8f048369273123981e6b66a5e3